docs/UserGuide.md

```diff
@@ -94,52 +75,52 @@ type fast, Socius can get your contact management tasks done faster than traditi
 | :wrench: | Help with common technical issues |
 
 --------------------------------------------------------------------------------------------------------------------
 
 ## Quick Start
 
-Here is a quick start on how you can start using our app in your own computer.
+Here is a quick start on how to use Socius on your computer.
 
-1. Ensure you have [*Java 11*](https://www.oracle.com/sg/java/technologies/javase/jdk11-archive-downloads.html) installed in your Computer.
+1. Install [*Java 11*](https://www.oracle.com/sg/java/technologies/javase/jdk11-archive-downloads.html) on your computer.
 
-2. Download the latest `Socius.jar` on our [Releases](https://github.com/AY2122S1-CS2103T-W08-4/tp/releases) page.
+2. Download the latest `Socius.jar` from [here](https://github.com/AY2122S1-CS2103T-W08-4/tp/releases).
 
-3. Move the file `Socius.jar` to a folder you want to use as the home folder for your Socius application.
+3. Move the `Socius.jar` file to a folder you want to use as the home folder for Socius.
 
-4. Double-click the file to start the app. A GUI similar to the one below should appear within a few seconds.
+4. Double-click the file to start Socius. A GUI similar to the one below should appear within a few seconds.
 
 <div markdown="block" class="alert alert-info">:wrench: **Help:**
-If double-click does not work, you can go to the terminal and type in `java -jar Socius.jar`. Ensure that the terminal is in the directory of the `Socius.jar` file.
+If double-clicking does not work, you can use the Terminal and type `java -jar Socius.jar`. Ensure that the Terminal is in the same directory as the `Socius.jar` file.
 </div>
 
    ![Ui](images/Quickstart.png)
 
-* Note that the app comes with some sample data by default. Type **`clear`** in the command box to remove those sample data.
+* Note that the app comes with some sample contacts by default. Type **`clear`** in the command box to remove those sample contacts.
 
-* Type the command in the command box and press `Enter` to execute it. e.g. typing **`help`** and pressing `Enter` will open the help window.
-Some example commands you can try:
-    * **`add`** `n/Amy Tan tg/W08 nat/Singaporean` : Adds a contact named `Amy Tan` to list, together with her tutorial group and nationality.
+* Type your command in the command box and press `Enter` to execute the command. e.g. typing **`help`** and pressing `Enter` will open the help window.
+These are some example commands that you can try:
+    * **`add`** `n/Amy Tan nat/Singaporean tg/W08` : Adds a contact named `Amy Tan`, a `Singaporean` in tutorial group `W08` to your contacts.
 
-    * **`delete`** `3` : Deletes the person at index 3 in the current list.
+    * **`delete`** `3` : Deletes the 3rd contact shown in the current list.
 
-    * **`find`** : `g/f nat/Singaporean` : Finds contacts of all female Singaporeans.
+    * **`find`** : `g/f nat/Singaporean` : Finds all female Singaporeans from your contacts.
```

Comment:
Some inconsistencies: Some parts female and Singaporeans would be in code format (i.e. `Female`, `Singaporean`)